src/sentry/integrations/slack/unfurl/discover.py

```diff
@@ -44,6 +45,9 @@ def unfurl_discover(data, integration, links: List[UnfurlableUrl]) -> UnfurledUr
         params = link.args["query"]
         query_id = params.get("id", None)
 
+        user_id = params.get("user", None)
+        user = User.objects.get(id=user_id) if user_id else None
```

Comment:
I don't think its a huge deal cause we still validate projects belong to org, but should we check that the user belongs to the org as well here?